containers/Dockerfile.microcosm

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.7
+MAINTAINER <>
+
+RUN apt-get -y update
+RUN apt-get -y install curl iproute2 netbase vim screen openssl
+
+ARG WEB_DOMAIN=dev.microco.sm
```

Comment:
Tweaking on yours will be difficult... I'll merge first, then fix up :)
Crazy what lax process we can have when a project is so new.